viscy/translation/engine.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self.l2_alpha = l2_alpha
         self.ms_dssim_alpha = ms_dssim_alpha
 
-    @torch.amp.custom_fwd(device_type="cuda", cast_inputs=torch.float32)
+    @torch.cuda.amp.custom_fwd(cast_inputs=torch.float32)
```

Comment:
Unrelated diff.

Reply:
Test will not pass otherwise. I can revert but tests will fail.

Reply:
https://github.com/mehta-lab/VisCy/pull/163#discussion_r1761572832

Reply:
I started the evaluation branch from the representation branch which does not have this change: https://github.com/mehta-lab/VisCy/blob/representation/viscy/translation/engine.py. How do I deal with this?

Reply:
Pull the base branch and rebase/merge based on your preference. Please consult [Git documentation](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) or search online.